src/main/java/org/tron/common/runtime/config/VMConfig.java

```diff
@@ -29,6 +32,8 @@
   private boolean vmTraceCompressed = false;
   private boolean vmTrace = false;
 
+  //Odyssey3.2 hard fork -- ForkBlockVersionConsts.ENERGY_LIMIT
+  public static boolean ENERGY_LIMIT_HARD_FORK = false;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make this "public static ENERGY_LIMIT_HARD_FORK" field final ([squid:S1444](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1444))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1690&issues=AWbxOpgATJjFgJLTP8Jd&open=AWbxOpgATJjFgJLTP8Jd)

[ISSUE_KEY]: AWbxOpgATJjFgJLTP8Jd